From a8ecc17025c4d3d72f9d5f1ce9d9a92417ccc92f Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Mon, 4 Mar 2019 19:44:12 +0100
Subject: [PATCH] TRITON-1276 sdcadm experimental update-gz-tools --latest
 --just-download dumps core Reviewed by: Trent Mick <trentm@gmail.com>
 Approved by: Trent Mick <trentm@gmail.com>

---
 lib/cli/do_update_gz_tools.js | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/lib/cli/do_update_gz_tools.js b/lib/cli/do_update_gz_tools.js
index 8baee6f..0cbdbee 100644
--- a/lib/cli/do_update_gz_tools.js
+++ b/lib/cli/do_update_gz_tools.js
@@ -509,7 +509,14 @@ UpdateGzTools.prototype.execute = function cExecute(opts, args, callback) {
                 next(err);
             });
         },
-        steps.servers.ensureServersRunning,
+
+        function ensureRunning(ctx, next) {
+            if (justDownload) {
+                next();
+                return;
+            }
+            steps.servers.ensureServersRunning(ctx, next);
+        },
 
         function downloadTarball(_, next) {
             if (filepath) {
-- 
2.21.0

